packages/email-pcd/src/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { EDdSAPublicKey } from "@pcd/eddsa-pcd";
+import { textToBigint } from "bigint-conversion";
+import { EmailPCD, IEmailData } from "./EmailPCD";
+
+/**
+ * One big int for each signed field in {@link IEmailData}
+ */
+export type SerializedEmailData = [bigint];
+
+export function emailDataToBigInts(data: IEmailData): SerializedEmailData {
+  return [textToBigint(data.emailAddress)];
+}
```

Comment:
I think you should sign the hash (represented by a big int) rather than using this `bigint-conversion` library. I recommend extracting the following function into the `@pcd/util` package, and then importing it here, and various other places we're generating hashes of strings and representing them as bigints.

https://github.com/proofcarryingdata/zupass/blob/rob/attested-email-feed/packages/semaphore-signature-pcd/src/SemaphoreSignaturePCD.ts#L39-L43

Reply:
I'm not aware of any other instances of hashing strings and representing them as bigints.

I used the ticket PCD as my model here, which simply converts all data to bigints. The EdDSAPCD package expects the message to be an array of bigints and it's the EdDSAPCD which takes those bigints, hashes, and signs them.